src/shell/dotnet/Shell/Fdc3/ResolverUi/RelayCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Morgan Stanley makes this available to you under the Apache License,
+ * Version 2.0 (the "License"). You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership. Unless required by applicable law or agreed
+ * to in writing, software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+using System;
+using System.Windows.Input;
+
+namespace MorganStanley.ComposeUI.Shell.Fdc3.ResolverUI;
+
+internal class RelayCommand : ICommand
```

Comment:
No need for this implementation. [RelayCommand](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/relaycommand) is already part of CommunityToolkit.Mvvm